packages/vhd-lib/src/vhd/abstractVhd.js

```diff
@@ -0,0 +1,100 @@
+import { computeBatSize } from './_utils'
+
+export class AbstractVhd {
```

Comment:
- `VhdAbstract`
  - `VhdDirectory`
  - `VhdFile`

Also, use the same casing for the filename, ie `VhdAbstract.js`.

Feel free to create a `Vhd` directory if you want to group these classes.